pkgs/tools/security/grap/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchFromGitHub, pkgs, python3 }:
```

Comment:
Please do not use pkgs as an input but list all packages individually.

Reply:
Ok.